benchs/nats-publisher/src/utils/tx.rs

```diff
@@ -84,11 +80,7 @@ impl TxHelper {
     ) -> anyhow::Result<()> {
         let tx_id = self.get_id(tx);
         let subject = self.get_subject(tx, block, index);
-        let payload = self
-            .nats
-            .data_parser()
-            .to_nats_payload(&subject.parse(), block)
-            .await?;
```

Comment:
Simplified encoding and decoding to remove the constraint on some payload structure. Now, `fuel-streams-core` houses the payload structure, `StreamData`, used for streaming which can be encoded and decoded by the `DataParser` with different strategies